### DISCUSSION TITLE
Development.Module vs Development

### DISCUSSION THREAD

**Discussion by richardwhitehead** [wjakob/nanobind#557] [Q&A]
I am a new user - this project is fantastic, thanks to all contributors!!
I followed the "tutorial" sections of the help (installing the library, setting up a build system, creating your first extension).
At first it failed to build, saying `Could NOT find Python (missing: Development.Module)`. 
`libpython3-dev` is installed and up to date.

Eventually I found the fix was to change `Development.Module` to `Development`. The project builds and works.

Being a new user I have no idea if this applies in some circumstances and not to others so I dare not propose a PR to change the docs!

I am on Ubuntu 22.04, in a virtual environment with Python3.8 installed (I am constrained to use this version).
Setting `-DPython_EXECUTABLE` was not required and made no difference.
I installed nanobind as a Git Submodule as described in the instructions (this saves having to modify a Yocto build for our target).

As I said, I have it working - it may just be worth either modifying the instructions or adding a note to the docs.
Thanks again for this excellent project.

**Comment by wjakob** [comment]
Explanation fixed in 5f6dc9a00ce6e160c5c378967ba9fe4318d9d6eb
